style/style.css

```diff
@@ -0,0 +1,152 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  background: url("https://web.archive.org/web/20140228114747im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg") 50% 100% no-repeat;
+  height: 68vh;
+}
```

Comment:
- [ ] In larger screens the body is not 100% wide. If you change the width you will see that some elements are not behaving the way they should and don't appear centered. I will comment on these things below in more detail.